VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/panel-model-base.ts

```diff
@@ -158,10 +158,12 @@ export abstract class PanelModel {
             return 1;
         } else if (this.role === Role.JudicialOfficeHolder) {
             return 2;
-        } else if (this.caseTypeGroup.toLowerCase() === 'endpoint') {
+        } else if (this.role === Role.QuickLinkParticipant) {
             return 4;
-        } else if (this.hearingRole === HearingRole.OBSERVER) {
+        } else if (this.caseTypeGroup.toLowerCase() === 'endpoint') {
             return 5;
+        } else if (this.hearingRole === HearingRole.OBSERVER || this.role === Role.QuickLinkObserver) {
+            return 6;
```

Comment:
This is for the main conference. Extending the way it currently works. Quick link participants go at the end of regular. And quicklink observers and regular observers are mixed